packages/runtime/src/runtime.ts

```diff
@@ -422,6 +427,37 @@ export class Runtime {
 		this.store.accounts.set(feeSink.address, feeSink);
 	}
 
+	/**
+	 * Add accounts from config file to the Store
+	 * @param network: the network of accounts to add
+	 * @param balance: balance for accounts
+	 */
+	loadAccountsFromConfig(network = NETWORK_DEFAULT, balance?: number): void {
+		const find = findupSync([JS_CONFIG_FILENAME, TS_CONFIG_FILENAME]);
+		if (!find) {
+			throw new RuntimeError(RUNTIME_ERRORS.GENERAL.CONFIG_FILE_NOT_FOUND);
+		}
+		// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
can we do it by `import as` instead of the require?

Reply:
`import as` should be use on the top of the file right. so using `require` here would be better than `import as`